ui/app/components/ui/popover/popover.component.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import PopoverHeader from './popover.header.component'
+
+const Popover = ({ title, children, onClose }) => (
+  <div className="popover-container">
+    <a href="#" className="popover-bg" onClick={onClose} />
```

Comment:
Why use an anchor for the background? A `div` would seem more appropriate. Using an anchor here is forcing the use of `preventDefault()` on the handler when it wouldn't otherwise be needed, and it's making the cursor think _the entire background_ is click-able, which seems wrong.